README.md

```diff
@@ -0,0 +1,9 @@
+# base-station
+
+A base station to run a rocket, and display some pretty graphs
+
+## How to run
+### Hardware code
+* Construct circuit shown in schematic
+* Move folders in ```hardware/src/libs``` to global Arduino library directory (usually ```Documents/Arduino/libraries```)
```

Comment:
Probably a good idea, but something to do on the side I guess as it's not really related to any scope